doc/hardware/porting/board_porting.rst

```diff
@@ -8,15 +8,69 @@ directory* with various files in it. Files in the board directory inherit
 support for at least one SoC and all of its features. Therefore, Zephyr must
 support your :term:`SoC` as well.
 
+.. _board_and_identifiers
+
+Board and board identifiers
```

Comment:
the intention was to cover the whole string under the term board identifier.

Meaning the board identifier captures to combined soc + cpu-cluster + variant identifier.

Would updating to:
`Board and identifiers` or
`Board and SoC / cpu-cluster / variant identifiers`

be better titles ?
I'm also open to other suggestions.

Reply:
Is there an overall "board identifier" concept? (something like "the collation of the soc/cpu-cluter/variant identifiers")?
(if there is it could just be described in the text / defined to be that in the latest paragraph of this section).

Reply:
A related topic, is the "underscory" version of the BOARD named something? (it could also be nice mentioning here that it exists, how it is generated, and that it is to automagically generate the board kconfig value)

Reply:
> Is there an overall "board identifier" concept? (something like "the collation of the soc/cpu-cluter/variant identifiers")?

Sort of, at least internally in the build system:
https://github.com/zephyrproject-rtos/zephyr/blob/85f99b0ae23822f96dac4c60cee79f9e7e214900/cmake/modules/boards.cmake#L77
https://github.com/zephyrproject-rtos/zephyr/blob/85f99b0ae23822f96dac4c60cee79f9e7e214900/boards/Kconfig.v2#L6

Just thinking that for the docs it might be valuable to not introduce to many new names.

> A related topic, is the "underscory" version of the BOARD named something? 

We have the generated `CONFIG_BOARD_<BOARD_FULL_STRING>` if that is what you're thinking about:
https://github.com/zephyrproject-rtos/zephyr/blob/85f99b0ae23822f96dac4c60cee79f9e7e214900/boards/Kconfig.v2#L13